### PR TITLE
[CHK-3264] feat: show continue to IO app button if redirect fail

### PR DESCRIPTION
**Pull request by scaminati-bv**
Depends on https://github.com/pagopa/pagopa-ecommerce-fe/pull/42

#### List of Changes
- add 2s delay to show _Continue to IO app_ button after redirect command

#### Motivation and Context
The goal of this PR is to prevent _Continue to IO app_ button from being shown for an instant even if the redirect goes successfully. If more than a second passes, it will mean that the redirect has failed and then that button message will be shown.

#### Screenshots (if appropriate):

https://github.com/user-attachments/assets/e0667dae-871e-487d-8721-27bdb1ba0fb6

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/utils/config/config.ts [hunk 1]
```diff
@@ -58,6 +59,12 @@ const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
         10
       )
     : 3000,
+  // eslint-disable-next-line no-underscore-dangle
+  ECOMMERCE_SHOW_CONTINUE_IO_BTN_DELAY_MILLIS: parseInt(
```
Comment (infantesimone) [inline src/utils/config/config.ts:63]: in order to make the code reliable, can you add a check if the value is present in configuration and configure a default value? like the previous parameters?

Comment (scaminati-bv) [reply src/utils/config/config.ts:63]: The fix consist of set a correct value for this variable like this, if env _ECOMMERCE_SHOW_CONTINUE_IO_BTN_DELAY_MILLIS_ is not present?

```
ECOMMERCE_SHOW_CONTINUE_IO_BTN_DELAY_MILLIS: parseInt(
    // eslint-disable-next-line no-underscore-dangle
    (window as any)._env_.ECOMMERCE_SHOW_CONTINUE_IO_BTN_DELAY_MILLIS,
    2000
 )
```

Comment (scaminati-bv) [reply src/utils/config/config.ts]: done here 053ee4464a4f3c63e1339730036a1b07a0d7cf4d

**Review by pietro-tota — APPROVED** [review]
🚀

**Pull request merged**